aws-predictions/src/main/java/com/amplifyframework/predictions/aws/AWSPredictionsPluginConfiguration.java

```diff
@@ -51,35 +55,51 @@ private AWSPredictionsPluginConfiguration(
      * @throws PredictionsException if configuration is missing or malformed
      */
     @NonNull
-    public static AWSPredictionsPluginConfiguration fromJson(JSONObject configurationJson) throws PredictionsException {
+    static AWSPredictionsPluginConfiguration fromJson(JSONObject configurationJson) throws PredictionsException {
         if (configurationJson == null) {
             throw new PredictionsException(
-                    "Could not locate predictions configuration for AWS Predictions Plugin.",
-                    "Verify that amplifyconfiguration.json contains a section for \"awsPredictionsPlugin\"."
+                "Could not locate predictions configuration for AWS Predictions Plugin.",
+                "Verify that amplifyconfiguration.json contains a section for \"awsPredictionsPlugin\"."
             );
         }
 
         final Region defaultRegion;
-        final AWSInterpretConfiguration interpretConfiguration;
+        final InterpretTextConfiguration interpretConfiguration;
+        final TranslateTextConfiguration translateTextConfiguration;
+
+        // Required sections
         try {
```

Comment:
I think that pattern would be good if manually editing the configuration file was common-place. However, I believe that what we should be doing is to encourage them to rely more on CLI and NOT have to worry about the configuration at all if possible. 

Besides, throwing on no-match sounds like it would make things difficult to leave comments on the configuration like this:
```json
{
  "__description": "NOT part of configuration, but helps me better identify what this section is about",
  "keyname1": "this is real key-value pair",
  "keyname2": "this is real key-value pair"
}
```

Reply:
I'm not sure what is an appropriate expectation regarding manual config vs. CLI. Definitely, the 90% happy path will be from folks doing an initial CLI setup.

The `"__description"` thing is more of a hack that I added into our config files. I have seen it used in other places. But it is not really common practice in JSON. 

Anyway, I don't think this is a major issue and I see your perspective on it, so I'm cool to let this one stay like it is!